src/obfl-specification.html

```diff
@@ -343,13 +343,52 @@ <h3 id="L5025">Auxiliary Flows</h3>
 of these containers, starting from 1 each time.</p>
 
 <h3 id="L5023">Simplex</h3>
-<p class="markup">In a sequence where duplex is not enabled, only the front side of sheets are counted and paginated. 
-The back side of sheets are neither counted nor paginated.</p>
+<p class="markup">In a sequence where duplex is not enabled, only the front side
+  of sheets are counted and paginated. The back side of sheets are neither
+  counted nor paginated.</p>
 
 <h3 id="L5024">Duplex</h3>
-<p class="markup">In a sequence where duplex is enabled, both sides of sheets are counted and paginated.
-However, if the last sheet of a duplex sequence only has content on the front side, 
-the back side is counted, but not <em>paginated</em>.</p>
+<p class="markup">In a sequence where duplex is enabled, both sides of sheets
+  are counted and paginated. However, if the last sheet of a duplex sequence
+  only has content on the front side, it depends on the values of the
+  attributes break-before and break-after on sequence, dynamic-sequence and
+  toc-sequence whether an extra page will be inserted so that the next sequence
+  starts at a right page, and whether that extra page is <em>paginated</em>
+  (which means that it has a page number and possibly a header and a footer).
+  The extra page is always counted. The default behavior is that an extra page
+  will be inserted which is not paginated.</p>
+
+<div class="markup">The just-mentioned break-before and break-after attributes
+  insert a break before or after the sequence, respectively. Depending on the
+  specific element, some of these values are supported:
+  <dl>
+    <dt>'volume'</dt>
+      <dd>inserts a volume break. This value only applies to text body
+        sequences.</dd>
+    <dt>'sheet'</dt>
+      <dd>inserts a sheet break. If this results in a page without content, it
+        is paginated.</dd>
+    <dt>'page'</dt>
+      <dd>inserts a page break. If there is a transition from duplex to
+        simplex or visa versa, or the page dimensions don't match, then perform
+        a sheet break. The 'page' value can be used to produce media without
+        blank pages.</dd>
+    <dt>'auto'</dt>
+      <dd>the default value 'auto' is like 'sheet' but leaving pages without
+        content completely blank (or absent).</dd>
+  </dl>
+</div>
+
+<p class="markup">The break-before and break-after attributes cover the cases
+  where you transition from the last sequence of the pre-content to the body
+  when the body is a resumption of a sequence in the previous volume. In cases
+  where a sequence is interrupted at the end of a volume and is followed by the
+  first sequence in the post-content, it is the break-before value of that new
+  sequence that matters. In cases where a sequence ends and is followed by a new
```

Comment:
This is quite literally taken from my GH issue. But my text was more to argue why we need two attributes. It's not so important for the reader to understand the reasoning behind it. I would shorten it as follows:

> When a sequence ends and is followed by a new sequence, both the break-after value of the sequence that ended and the break-before value of the new sequence must be taken into account: 'volume' dominates over all the other values, 'sheet' dominates over 'page' and 'auto', and 'page' dominates over 'auto'. Note that a sequence does not 'end' when it is interrupted by a series of post-content and/or pre-content sequences (i.e. at a volume break), and likewise it does not 'start' when it is resumed in the next volume.

I think this is slightly more to the point.

Reply:
Done.